### PR TITLE
Update README with vscode instructions

### PR DESCRIPTION
**Pull request by ranjitjhala**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,3 +100,33 @@ $ cargo test -p flux-tests
 ## Limitations
 
 This is a prototype! Use at your own risk. Everything could break and it will break.
+
+
+## Rust-Analyzer in VSCode
+
+Add this to the workspace settings popping in the appropriate paths for 
+
+* `DYLD_FALLBACK_LIBRARY_PATH`
```
Comment (nilehmann) [inline README.md:109]: We should add that for Linux this is one is `LD_LIBRARY_PATH`

File: README.md [hunk 2]
```diff
@@ -100,3 +100,33 @@ $ cargo test -p flux-tests
...
+* `RUSTC_WRAPPER` 
+* `RUSTUP_TOOLCHAIN`
+
+```
+{
+  "rust-analyzer.checkOnSave.overrideCommand": [
+    "env",
+    "DYLD_FALLBACK_LIBRARY_PATH=/Users/rjhala/.rustup/toolchains/nightly-2022-08-02-x86_64-apple-darwin/lib",
+    "RUSTC_WRAPPER=/Users/rjhala/research/rust/flux/target/debug/flux",
+    "RUSTUP_TOOLCHAIN=nightly-2022-08-02",
```
Comment (nilehmann) [inline README.md:119]: I'd hint that this value comes from `rust-toolchain.toml`

**Review by nilehmann — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -100,3 +100,33 @@ $ cargo test -p flux-tests
...
+    "check",
+    "--message-format=json"
+  ],
+  "spellright.language": [
+    "en"
+  ],
+  "spellright.documentTypes": [
+    "latex",
+    "plaintext"
+  ]
```
Comment (nilehmann) [inline README.md:130]: this is not related to flux

**Review by nilehmann — CHANGES_REQUESTED** [review]
Actually, now I realize your configuration is not the proper way to do it anymore. The following is the better way now


```json
{
  "rust-analyzer.checkOnSave.extraEnv": {
    "RUSTC_WRAPPER": "/home/nlehmann/phd/liquid-rust/flux/target/release/flux",
    "RUSTUP_TOOLCHAIN": "nightly-2022-08-02",
    "LD_LIBRARY_PATH": "/home/nlehmann/.rustup/toolchains/nightly-2022-08-02-x86_64-unknown-linux-gnu/lib"
  }
}
```

**Comment by ranjitjhala** [comment]
hey can we merge this?
